README.md

```diff
@@ -663,8 +796,8 @@ A revocation message MUST conform to the following format:
 ``` js
 {
   "iss": did,
-  "revoke": ucanCid,
-  "challenge": sign(did.privateKey, `REVOKE:${ucanCid}`)
+  "revoke": canonicalUcanCid,
+  "challenge": sign(did.privateKey, `REVOKE:${canonicalUcanCid}`)
```

Comment:
how is this json-encoded? base64? multibase?

Reply:
Good point! base64 👍 I'll add that to the text